draft-iab-privacy-partitioning.md

```diff
@@ -623,15 +623,19 @@ If the Oblivious Relay and Gateway collude, they can link Client identity and da
 for each request and response transaction by simply observing requests in transit.
 
 It is not currently possible to guarantee with technical protocol measures that two
-entities are not colluding. However, there are some mitigations that can be applied
+entities are not colluding. Further even if entities do not collude directly, if information
+is revealed to another party, there might be no control about further collusion of data.
+However, there are some mitigations that can be applied
```

Comment:
```suggestion
entities are not colluding. Even if two entities do not collude directly, if both entities reveal
information to other parties, it will not be possible to guarantee that the information won't
be combined. However, there are some mitigations that can be applied
```